src/components/QueryRecordings/QueryRecordings.vue

```diff
@@ -266,7 +266,8 @@ export default {
   mounted() {
     this.isAudio = this.recordingType === "audio";
     this.parseCurrentRoute();
-    this.updateRouteQuery();
+    this.lastQuery = JSON.parse(JSON.stringify(this.query));
```

Comment:
This should be pulled out into it's own method (saveLastQuery or similar) since it's reasonably involved and now done in multiple places.